gulpfile.js

```diff
@@ -362,16 +359,17 @@ gulp.task('karma:coverage', function(done) {
 // Drops the MongoDB database, used in e2e testing
 gulp.task('dropdb', function (done) {
   // Use mongoose configuration
-  var mongoose = require('./config/lib/mongoose.js');
+  var mongooseService = require('./config/lib/mongoose');
```

Comment:
If our Mongoose service is only being used in a couple places, why `require` it at the global scope?

My understanding of how `require` works, along with it's memory use, isn't great so maybe it doesn't matter.